04/SpeedTest.js

```diff
@@ -1,17 +1,52 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState, useRef } from 'react';
 import useRandomItem from './hook';
 
+import DisplayResults from './DisplayResults'
+
 const SpeedTest = () => {
     const [word, regenerateWord] = useRandomItem(['devmentor.pl', 'abc', 'JavaScript']);
+    const [enteredText, setEnteredText] = useState('');
+    const [seconds, setSeconds] = useState(0);
+    const [results, setResults] = useState([])
+
+    const clockRef = useRef(null);
+    const inputRef = useRef(null);
+
+    const startClock = () => {
+        clockRef.current = setInterval(() => {
+            setSeconds(value => value + 1);
+        }, 1000);
```

Comment:
W tym przypadku sekundy to zbyt długi odstęp czasu.
Proponuję go zmniejszyć i dodać timer na stronie ;P